packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -63,6 +61,23 @@ export const CaptionsBanner = (props: { isMobile: boolean }): JSX.Element => {
   const captionsBannerStrings: _CaptionsBannerStrings = {
     captionsBannerSpinnerText: strings.captionsBannerSpinnerText
   };
+  /* @conditional-compile-remove(close-captions) */
+  const { innerWidth: width } = window;
+  /* @conditional-compile-remove(close-captions) */
+  const [windowWidth, setWindowWidth] = useState(width);
```

Comment:
Is it possible to use window.innerWidth in this useState?
I'm guessing it's not cause you have line 65.